utils/ci-test.sh

```diff
@@ -1,31 +1,42 @@
 #!/bin/bash
 
+# SKIP_LINT=true - only run the tests
+
 set -e
 
+MATRIX=(
+	""
+	"-tags coprocess"
+)
+
 # print a command and execute it
 show() {
-	echo "$@"
+	echo "$@" >&2
```

Comment:
Well stderr for errors, stdout for warns and info messages. Just about semantics. 
In this concrete case, there is high chance that Travis renders "stderr" and "stdout" with different color, so there may be some confusion.